install/kubernetes/helm/istio/files/injection-template.yaml

```diff
@@ -453,6 +453,13 @@ dnsConfig:
     {{- end }}
 {{- end }}
 podRedirectAnnot:
+{{- if and (.Values.istio_cni.enabled) (ne .Values.istio_cni.istio_cni_network "") }}
+{{ if isset .ObjectMeta.Annotations `k8s.v1.cni.cncf.io/networks` }}
+   k8s.v1.cni.cncf.io/networks: "{{ index .ObjectMeta.Annotations `k8s.v1.cni.cncf.io/networks`}}, {{.Values.istio_cni.istio_cni_network}}"
```

Comment:
just out of curiosity: is the order important here? in my fix I had this the other way around

Reply:
I believe it is applied in order so the Istio CNI should be last to run